### DISCUSSION TITLE
Is there any way to change font colors in profile README

### DISCUSSION THREAD

**Discussion by hoosiermate** [community/community#19188] [Other Feature Feedback, Questions, & Ideas] [answered]
I think it was possible a long time ago, but not anymore?

**Comment by justin-2028** [comment, marked as answer]
Don't think it's possible anymore, but check these articles for other ways of customizing your profile:
https://medium.com/analytics-vidhya/writing-github-readme-e593f278a796
https://dev.to/kyechan99/make-your-profile-readme-colorful-3gag

gl
